interactive_color.py

```diff
@@ -0,0 +1,751 @@
+                pygame.quit()
+                return
+
+    # View-------------------------------------------------------------
+    # prints the background
+    screen.fill(WHITE)
+
+    # This prints all of the blocks on the screen
+    # row and column counts are used to keep track of matrix row and column
+    # there is a for loop to run through each row and column
+    row_count = -1
+    for row in field.matrix:
+        column_count = -1
+        row_count += 1
+        for column in row:
+            column_count += 1
+            if field.matrix[row_count][column_count] != 0:
+                if field.matrix[row_count][column_count] == 4:
+                    # based on the number entry in the matrix, it prints a different block
+                    screen.blit(spring, (column_count*40, row_count*40))
+                if field.matrix[row_count][column_count] == 1:
+                    screen.blit(grass, (column_count*40, row_count*40))
+                if field.matrix[row_count][column_count] == 2:
+                    screen.blit(dirt, (column_count*40, row_count*40))
+                if field.matrix[row_count][column_count] == 3:
+                    screen.blit(stone, (column_count*40, row_count*40))
+                if field.matrix[row_count][column_count] == 9:
+                    screen.blit(bedrock, (column_count*40, row_count*40))
+    inventory.draw_inventory(field, inventory_block_index, grass, stone, dirt, bedrock, spring)
+    player.draw(amon_picture, sean, colvin)
+    enemy1.draw(zombie)
+    return [level_select, inventory_block_index, previous_level_select, player_color, done]
+
+
+# Control
+def main():
+    """ The actual call of the start of the program.
+    Main movement is referenced by this function.
+    """
+    # how long between each cycle of the while loop
+    clock = pygame.time.Clock()
+
+    # initializes player, field, and other variables for level one and two
+    player_color = 0
+    player_color2 = 0
+    previous_level_select = "unknown"
+    level_select = "Menu"
+    done = False
+    player = Player()
+    field = Field()
+    field2 = Field()
+    field2.matrix = level_two_map()
+    player2 = Player()
+    player2.x = 0
+
+    enemy1 = Enemy()
+    inventory = Inventory(0, 0, 20, 40, 40)
+    inventory2 = Inventory(0, 0, 20, 40, 40)
+    inventory_block_index = 1
+    inventory_block_index2 = 1
+
+    # loads all the pictures
+    amon_picture = pygame.image.load('amon.png')
+    grass = pygame.image.load("grass.png")
+    stone = pygame.image.load("stone.png")
+    dirt = pygame.image.load("dirt.png")
+    soulsand = pygame.image.load("soulsand.png")
+    netherack = pygame.image.load("netherack.png")
+    netherquartz = pygame.image.load("netherquartz.png")
+    bedrock = pygame.image.load("bedrock.png")
+    spring = pygame.image.load("spring.png")
+    sean = pygame.image.load("sean.png")
+    colvin = pygame.image.load("colvin.png")
+    zombie = pygame.image.load("Zombie.png")
```

Comment:
Encapsulate all these loading commands into `image_load()` or some kind of setup function?